addons/l10n_ec/data/template/account.tax-ec.csv

```diff
@@ -1,349 +1,349 @@
+"tax_vat_511","12% A","purchase","percent","10","12.0","VAT 12% (assets)","12%","511","521","tax_group_vat_12","","base","invoice","+511 (Reporte 104)","","","Iva 12% (activos)"
 "","","","","","","","","","","","","tax","invoice","+521 (Reporte 104)","ec_purchase_vat_assets","",""
 "","","","","","","","","","","","","base","refund","-511 (Reporte 104)","","",""
 "","","","","","","","","","","","","tax","refund","-521 (Reporte 104)","ec_purchase_vat_assets","",""
-"tax_vat_512","VAT 12% (no tax credit)","purchase","percent","10","12.0","","VAT 12%","512","522","tax_group_vat_12","","base","invoice","+512 (Reporte 104)","","","Iva 12% (sin crédito tributario)"
+"tax_vat_512","12% NT","purchase","percent","10","12.0","VAT 12% (no tax credit)","12%","512","522","tax_group_vat_12","","base","invoice","+512 (Reporte 104)","","","Iva 12% (sin crédito tributario)"
 "","","","","","","","","","","","","tax","invoice","+522 (Reporte 104)","","",""
 "","","","","","","","","","","","","base","refund","-512 (Reporte 104)","","",""
 "","","","","","","","","","","","","tax","refund","-522 (Reporte 104)","","",""
-"tax_vat_513","VAT 12% (import of services)","purchase","percent","10","12.0","","VAT 12%","513","523","tax_group_vat_12","","base","invoice","+513 (Reporte 104)","","","Iva 12% (importación servicios)"
+"tax_vat_513","12% IMP S","purchase","percent","10","12.0","VAT 12% (import of services)","12%","513","523","tax_group_vat_12","","base","invoice","+513 (Reporte 104)","","","Iva 12% (importación servicios)"
 "","","","","","","","","","","","","tax","invoice","+523 (Reporte 104)","ec_purchase_vat_service_imports","",""
 "","","","","","","","","","","","","base","refund","-513 (Reporte 104)","","",""
 "","","","","","","","","","","","","tax","refund","-523 (Reporte 104)","ec_purchase_vat_service_imports","",""
-"tax_vat_514","VAT 12% (import of goods)","purchase","percent","10","12.0","","VAT 12%","514","524","tax_group_vat_12","","base","invoice","+514 (Reporte 104)","","","Iva 12% (importación bienes)"
+"tax_vat_514","12% IMP G","purchase","percent","10","12.0","VAT 12% (import of goods)","12%","514","524","tax_group_vat_12","","base","invoice","+514 (Reporte 104)","","","Iva 12% (importación bienes)"
 "","","","","","","","","","","","","tax","invoice","+524 (Reporte 104)","ec_purchase_vat_goods_imports","",""
 "","","","","","","","","","","","","base","refund","-514 (Reporte 104)","","",""
 "","","","","","","","","","","","","tax","refund","-524 (Reporte 104)","ec_purchase_vat_goods_imports","",""
-"tax_vat_515","VAT 12% (import assets)","purchase","percent","10","12.0","","VAT 12%","515","525","tax_group_vat_12","","base","invoice","+515 (Reporte 104)","","","Iva 12% (importación activos)"
+"tax_vat_515","12% IMP A","purchase","percent","10","12.0","VAT 12% (import assets)","12%","515","525","tax_group_vat_12","","base","invoice","+515 (Reporte 104)","","","Iva 12% (importación activos)"

```

Comment:
It seems like this fix is not using the new l10n_ec module made for v16.0 and merged just in march, the names were already improved a bit:
```suggestion
"tax_vat_515","12% IMP A","purchase","percent","10","12.0","VAT 12% (import assets)","12%","515","525","tax_group_vat_12","","base","invoice","+515 (Reporte 104)","","","IVA 12% (Importación Activos)"

```